packages/mui/PUBLISHING.md

```diff
@@ -14,7 +14,7 @@ If you need to publish a package manually, you can run the following commands fr
 
 ```sh
 yarn build
-cd mui && yarn build
+cd mui/dist
```

Comment:
These instructions should be relative to the root folder of the package...so:
```
cd dist
```
instead of
```
cd mui/dist
```